testing/eolie/APKBUILD

```diff
@@ -0,0 +1,36 @@
+# Contributor: Daniele Debernardi <drebrez@gmail.com>
+# Maintainer: Daniele Debernardi <drebrez@gmail.com>
+pkgname=eolie
+pkgver=0.9.63
+pkgrel=0
+pkgdesc="Web browser for GNOME"
+url="https://gitlab.gnome.org/World/eolie"
+arch="all"
+license="GPL-3.0-or-later"
+depends="webkit2gtk gtkspell3 py3-cairo py3-gobject3"
+makedepends="meson gobject-introspection-dev webkit2gtk-dev python3-dev py3-gobject3-dev"
+source="$pkgname-$pkgver.tar.xz::https://gitlab.gnome.org/World/eolie/uploads/d42b466752729a0d3fa828a721e25043/eolie-0.9.63.tar.xz
```

Comment:
Please add `desktop-file-utils` and `appstream-glib` as ` checkdepends`, they're used for additional tests.

```
...
Message: Compiling resources
Configuring AboutDialog.ui using configuration
Found pkg-config: /usr/bin/pkg-config (1.6.3)
Program desktop-file-validate found: NO
Program appstream-util found: NO
Program glib-compile-schemas found: YES (/usr/bin/glib-compile-schemas)
Message: Making a list of icons
...
```

Reply:
Added 👍